src/puppetlabs/puppetdb/import.clj

```diff
@@ -12,12 +12,36 @@
             [puppetlabs.puppetdb.export :as export]
             [puppetlabs.puppetdb.utils :as utils]
             [puppetlabs.puppetdb.cheshire :as json]
-            [puppetlabs.puppetdb.schema :refer [defn-validated]]))
+            [puppetlabs.puppetdb.schema :refer [defn-validated]]
+            [puppetlabs.puppetdb.utils :as utils]))
 
 (defn file-pattern
   [entity]
   (re-pattern (str "^" (.getPath (io/file utils/export-root-dir entity ".*\\.json")) "$")))
 
+(def after-certname-length
+  "The format is <certname>-<date hash><suffix>. This is a count of
+  the characters after the certname"
+  (+ 1
+     40
+     (count export/file-suffix)))
```

Comment:
realistically, how much value is there in putting file-suffix in a global variable? Are we ever going to do anything but .json for this?